apps/theme/pageBuilder/styles/medium.scss

```diff
@@ -1,8 +1,8 @@
 /**
  *  `Medium editor` styles
  */
-@import "~medium-editor/dist/css/medium-editor.css";
-@import "~medium-editor/dist/css/themes/mani.css";
+@import "medium-editor/dist/css/medium-editor.css";
```

Comment:
Existing user projects will still have imports via `~`. Will this upgrade break this for those projects, or is this backwards compatible? This has to be tested with older projects.

Reply:
It's being deprecated but I couldn't find any notification as to when they might remove it. https://webpack.js.org/loaders/sass-loader/#resolving-import-at-rules